checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstylePlugin.java

```diff
@@ -57,6 +57,14 @@
             + "SuppressionCommentFilter</a>: "
             + FILTERS_DESCRIPTION_FOOTER;
 
+    private static final String CHECKER_TAB_WIDTH_DESCRIPTION = FILTERS_DESCRIPTION_HEADER
```

Comment:
>FILTERS_DESCRIPTION_HEADER

Please rename this variable to smth more generic

Reply:
changed as well.